src/client/stories/suggestedRoutes/suggestedRouteCard.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import SuggestedRouteCard from "../../components/SuggestedRoutes/SuggestedRouteCard";
+import {
+  withKnobs,
+  text,
+  boolean,
+  number,
+  array,
+  object
+} from "@storybook/addon-knobs";
+
+export function suggestedRouteCard() {
+  return (
+    <SuggestedRouteCard
+      time={text("theTime")}
+      price={text("thePrice")}
+      eta={text("ETA")}
+      sailDays={text("SAIL DAYS")}
+      maxWaveHs={text("MAX WAVE HS")}
+      avgSpeed={text("AVG SPEED")}
+      hireCost={text("HIRE COST")}
+      consumption={text("CONSUMPTION")}
```

Comment:
A small thing:

These are the labels you see in Storybook. It's not that important, but to satisfy my OCD it would be nice if you will make the labels a little more consistent:
```suggestion
time={text("Time")}
price={text("Price")}
eta={text("ETA")}
sailDays={text("Sail days")}
maxWaveHs={text("Max wave HS")}
avgSpeed={text("Average speed")}
hireCost={text("Hire cost")}
consumption={text("Consumption")}
```

Reply:
Note that it is only abbreviations that are capitalized.

Reply:
@zkwsk I didn't understand what should I do exactly!

Reply:
Done ;)